practice/src/css/bases/typography.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Barlow";
+  src: url("../../assets/fonts/Barlow-SemiBold.woff") format("woff");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "Barlow";
+  src: url("../../assets/fonts/Barlow-Medium.woff") format("woff");
+  font-display: swap;
```

Comment:
could you explain to me why we need `font-display: swap;`?

Reply:
Add `font-display: swap` for typography to improve fonts loading speed. It was suggested when I was testing pagespeed in html-css-practice

Reply:
It's not correct, it can not impove fonts loading speed :)

```
swap: Instructs the browser to use the fallback font to display the text until the custom font has fully downloaded. This means the text is shown immediately in the best available font, then swapped to the custom font once it's ready.
```